lib/unison-pretty-printer/tests/Unison/Test/ColorText.hs

```diff
@@ -7,7 +7,7 @@ module Unison.Test.ColorText where
 import qualified Data.Map as Map
 import EasyTest
 import Text.RawString.QQ
-import Unison.Lexer (Pos (..))
+import Unison.Lexer.Pos (Pos (..))
```

Comment:
`Pos` used to be defined in the `Unison.Lexer` module. I split it out to it's own module to eliminate unhelpful dependencies on `Unison.Lexer`, but left a forwarding export there. We can rename the `Pos` module to whatever though, moving forward.